test/regression.py

```diff
@@ -1810,12 +1823,40 @@ class ADFSPTest_nosyms(ADFSPTest, GenericSPTest):
     def testsymlabels(self):
         """Symmetry labels were not printed here."""
 
+class ADFSPTest_nosyms_noscfvalues(ADFSPTest_nosyms):
+    @unittest.skip('Cannot parse scfvalues from this file.')
+    def testscftargetdim(self):
+        """SCF cycles were not printed here."""
+
+    @unittest.skip('Cannot parse scfvalues from this file.')
+    def testscfvaluetype(self):
+        """SCF cycles were not printed here."""
+
+    @unittest.skip('Cannot parse aooverlaps from this file.')
+    def testaooverlaps(self):
+        """AO overlaps were not printed here."""
+
 class ADFSPTest_nosyms_valence(ADFSPTest_nosyms):
     def testlengthmoenergies(self):
         """Only valence orbital energies were printed here."""
         self.assertEqual(len(self.data.moenergies[0]), 45)
         self.assertEqual(self.data.moenergies[0][0], 99999.0)
 
+class ADFSPTest_nosyms_valence_noscfvalues(ADFSPTest_nosyms_valence):
+    @unittest.skip('Cannot parse scfvalues from this file.')
+    def testscftargetdim(self):
+        """SCF cycles were not printed here."""
+
+    @unittest.skip('Cannot parse scfvalues from this file.')
+    def testscfvaluetype(self):
+        """SCF cycles were not printed here."""
+
+    @unittest.skip('Cannot parse aooverlaps from this file.')
+    def testaooverlaps(self):
+        """AO overlaps were not printed here."""
+

```

Comment:
Let's follow PEP8 and surround class definitions with two blank lines: https://www.python.org/dev/peps/pep-0008/#blank-lines

Reply:
No problem! Thank you for the link. I have made these changes for the classes I've introduced into this file. Would you like me to change this for all classes in the regression.py file?

Reply:
Sure, if you're up for it.